src/main/resources/static/main.css

```diff
@@ -82,6 +82,10 @@ i{
 .toggle>.toggle-group>.toggle-off,.toggle>.toggle-group>.toggle-on{
     padding-left:25px
 }
+.toggle[disabled]{
```

Comment:
let's change the `cursor` from `pointer` to `not-allowed`

Reply:
Why?

Reply:
@jugendhacker And let's change the `background-color` to `#AAAAAA`

Reply:
while the button is disabled there is no point of keeping `cursor` as `pointer`

Reply:
Done